packages/components/organisms/f-form/src/components/Form.vue

```diff
@@ -1,46 +1,102 @@
 <template>
-    <div
+    <form
+        method="post"
         :class="$style['c-form']"
-        data-test-id="form-component">
-        {{ copy.text }}
-    </div>
+        data-test-id="form-component"
+        @submit.prevent="onFormSubmit">
+        <form-field
+            v-for="field in formData.formFields"
+            :key="`${field.name}-field`"
+            v-bind="fieldProps(field)"
+            @input="updateField({ fieldName: field.name, value: $event })" />
+
+        <f-button
+            button-type="primary"
+            button-size="large"
+            :class="$style['c-form-submitButton']"
+            is-full-width
+            action-type="submit"
+            data-test-id="submit-button"
+            :is-loading="isFormSubmitting">
+            {{ formData.buttonText }}
```

Comment:
I'd probably have a sensible default `buttonText` in case someone doesn't pass one in for some reason (for example, `Submit`).